src/cart/crt_internal_fns.h

```diff
@@ -42,8 +42,8 @@ void crt_req_timeout_untrack(struct crt_rpc_priv *rpc_priv);
 void crt_req_force_timeout(struct crt_rpc_priv *rpc_priv);
 
 /** crt_hlct.c */
-uint64_t crt_hlct_get(void);
-void crt_hlct_sync(uint64_t msg);
+uint64_t d_hlct_get(void);
+void d_hlct_sync(uint64_t msg);
```

Comment:
Should be coming from gurt/common.h now right?

Reply:
yup, removed.